researchstack-mpower/src/main/java/org/sagebionetworks/research/mpower/researchstack/framework/MpTaskHelper.java

```diff
@@ -69,7 +69,11 @@
     /**
      * The RESULT_CONVERSION_MAP allows you to map any step identifier result to whatever you want
      */
-    static HashMap<String, String> RESULT_CONVERSION_MAP = new HashMap<>();
+    static HashMap<String, String> RESULT_CONVERSION_MAP = new HashMap<String, String>() {{
+        put("hr_rgb.json_hr", "hr_rgb");
+        put("accel_stairStep", "motion");
+        put("peak", "samples");
+    }};
 
     public MpTaskHelper(final StorageAccessWrapper storageAccess,
```

Comment:
This was an interesting bug that I found.  The MpTaskHelper was included in the code, but it wasn't actually linked up in the ResearchStack initialization code.  Once I linked it up, I was able to do identifier substitution to match iOS' schema for the Heart Snapshot.